kafka-commons/src/test/java/uk/gov/homeoffice/digital/sas/kafka/serializer/CustomJsonSerializerTest.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.homeoffice.digital.sas.kafka.serializer;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static uk.gov.homeoffice.digital.sas.kafka.serializer.CustomJsonSerializer.createMapper;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Date;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.SpringBootConfiguration;
+import uk.gov.homeoffice.digital.sas.model.Profile;
+import uk.gov.homeoffice.digital.sas.utils.TestUtils;
+
+@SpringBootConfiguration
+class CustomJsonSerializerTest {
+
+  private static final Long PROFILE_ID = 1L;
+  private static final String TENANT_ID = "tenantId";
+  private static final String PROFILE_NAME = "Original profile";
+  private final static Date START_TIME = TestUtils.getAsDate(LocalDateTime.of(
+      2022, 04, 17, 01, 30));
+
+  private Profile profile;
+
+  @Test
+  void whenObjectMapperCreated_canDeserializeDateFields() {
+
+    ObjectMapper objectMapper = createMapper();
```

Comment:
As this mapper is used in both tests, it would be sensible to move it at test class level

Reply:
Move to class level